static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -0,0 +1,120 @@
+import {CSSProperties} from 'react';
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/featureBadge';
+import FeedbackItemUsername from 'sentry/components/feedback/feedbackItem/feedbackItemUsername';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import Link from 'sentry/components/links/link';
+import {Flex} from 'sentry/components/profiling/flex';
+import TextOverflow from 'sentry/components/textOverflow';
+import TimeSince from 'sentry/components/timeSince';
+import {IconPlay} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {
+  FeedbackItemLoaderQueryParams,
+  HydratedFeedbackItem,
+} from 'sentry/utils/feedback/types';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+interface Props {
+  feedbackItem: HydratedFeedbackItem;
+  className?: string;
+  style?: CSSProperties;
+}
+
+const ReplayBadge = styled(props => (
+  <span {...props}>
+    <IconPlay size="xs" />
+    {t('Replay')}
+  </span>
+))`
+  display: flex;
+  gap: ${space(0.5)};
+  align-items: center;
+`;
+
+function UnreadBadge() {
+  return <FeatureBadge type="new" variant="indicator" />;
```

Comment:
Damnit just clicked into this and it cleared the red indicator, but wondering how many other badges we have in use that are not `FeatureBadge`:

![image](https://github.com/getsentry/sentry/assets/79684/666bd310-775b-4fed-b5a7-e5a550e95273)

Reply:
We have the plain `<Badge>` in a couple spots

It's similar and more flexible than FeatureBadge, so easy to use instead: https://sentry.sentry.io/stories/?name=app/components/badge.stories.tsx